pkgs/development/tools/pyre/default.nix

```diff
@@ -33,6 +33,7 @@ in stdenv.mkDerivation {
     ppx_deriving_yojson
     ocamlbuild
     ppxlib
+    # python36Packages.python36Full # TODO
```

Comment:
Can you expand the TODO with a closing condition? I'm not sure what needs to be done to resolve the TODO.

Reply:
Can do, this is just for when we add the python client, `pyre`.

Reply:
Cool - LGTM after additional comment or removal. Thanks for the update!